CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ project adheres to [Semantic Versioning](http://semver.org/).
 - chore: add test for `process_start_time_seconds`
 
 ### Added
+- `nodejs_gc_runs` metric to the `collectDefaultMetrics()`. It counts number of GC runs with split by GC type.
```

Comment:
Unnecessary duplicate metric? The histogram contains the counts, so a seperate metrics is a waste of storage. gc counts (and rates) can be graphed from the histogram.

Reply:
yes, you are right.  Histogram already has nodejs_gc_duration_seconds_count metric with the same information